mappings/com/mojang/blaze3d/platform/GlStateManager.mapping

```diff
@@ -286,6 +287,24 @@ CLASS net/minecraft/class_4493 com/mojang/blaze3d/platform/GlStateManager
 		ARG 1 access
 	METHOD method_31947 unmapBuffer (I)V
 		ARG 0 target
+	METHOD method_34404 bindAttribLocation (IILjava/lang/CharSequence;)V
+		ARG 0 program
+		ARG 1 index
+		ARG 2 name
+	METHOD method_34405 shaderSource (ILjava/util/List;)V
+		ARG 0 shader
+		ARG 1 strings
+	METHOD method_34406 vertexAttribIPointer (IIIIJ)V
```

Comment:
Is this supposed to have the `I` between `Attrib` and `Pointer`

Reply:
I'm not sure, but it's not a typo by me at least. That `I` comes from the LWJGL/OpenGL method name (`GL30.glVertexAttribIPointer`).

Reply:
It's an [OpenGL API function](https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glVertexAttribPointer.xhtml), so not a LWJGL typo either. (Curiously, there's also a variant without the `I`)